Content.Server/Nutrition/Components/FoodComponent.cs

```diff
@@ -81,12 +81,12 @@ public sealed partial class FoodComponent : Component
     /// <summary>
     ///     Frontier - Nasty food, used for goblins to know if they can eat it or not
     /// </summary>
-    [ViewVariables(VVAccess.ReadWrite), DataField] // Frontier
+    [DataField, ViewVariables] // Frontier
```

Comment:
The default ViewVariables value is not ReadWrite, it is ReadOnly.

Source: Robust.Shared.ViewVariablesAttribute.cs:12